erpnext/accounts/doctype/pricing_rule/utils.py

```diff
@@ -67,13 +67,12 @@ def sorted_by_priority(pricing_rules, args, doc=None):
 			if not pricing_rule.get('priority'):
 				pricing_rule['priority'] = 1
 
-			if pricing_rule.get('apply_multiple_pricing_rules'):
```

Comment:
What was the significance of this parameter earlier, completely removing this won't cause any issues in other scenarios? Isn't there a checkbox in the form corresponding to this parameter, will it be still useful?

Reply:
`sorted_by_priority` should only sort the `pricing_rules` by priority. 
Whether multiple pricing rules are to be applied or not, should be decided at some other place. 

This should not break anything since `sorted_by_priority` is only called if (line 45) `apply_multiple_pricing_rules(pricing_rules)` is `True`

> Isn't there a checkbox in the form corresponding to this parameter, will it be still useful?
Yes, that check is already being used in `apply_multiple_pricing_rules()`
Also, this check can only be enabled from promotional schemes, I guess.

Reply:
After discussing with @rohitwaghchaure, found out that this change is not needed. Hence removing it.